setup.py

```diff
@@ -23,7 +23,8 @@
     'ipdb',
     'coveralls',
     'sphinx',
-    'vcrpy'
+    'vcrpy',
+    'multicache'
```

Comment:
multicache won't be needed here, since it is specified in the install_requires section